### PR TITLE
Implement payload size metrics

### PR DESCRIPTION
**Pull request by ctron**
Records the payload size for MQTT and HTTP messages by measuring the payload body.

### CODE REVIEW

**Comment by ctron** [comment]
> Don't forget to also add corresponding templates to example/src/main/deploy/influxdb.conf

@sophokles73  Good point. I did add an entry, however my understanding of how this mapping really works is rather vague. Maybe you could double check it. Thanks.

File: adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HttpAdapterMetrics.java [hunk 1]
```diff
@@ -36,4 +37,13 @@ void incrementUndeliverableHttpMessages(final String resourceId, final String te
         counterService.increment(getPrefix() + MESSAGES + mergeAsMetric(resourceId,tenantId) + UNDELIVERABLE);
     }
 
+    void incrementProcessedHttpPayload(final String resourceId, final String tenantId, final long payloadSize) {
```
[the remark this answers is not in the document]
Comment (ctron) [reply adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HttpAdapterMetrics.java]: Well I did align that with the other methods. So yes, that can all be pulled up I guess.

[the remark this answers is not in the document]
Comment (sophokles73) [reply adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HttpAdapterMetrics.java]: @ctron are you going to pull up the methods?

[the remark this answers is not in the document]
Comment (ctron) [reply adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HttpAdapterMetrics.java]: I was waiting for a confirmation from your side :wink: … yes, I will do that now.

**Comment by ctron** [comment]
Ok, I rebased and pulled up the common methods.

**Pull request merged**